src/client/components/ReviewAndSubmitSection.jsx

```diff
@@ -45,28 +45,28 @@ class ReviewAndSubmitSection extends React.Component {
 
         {/* TODO(crew): Change names of sections to real names. */}
         <ReviewCollapsiblePanel
-            sectionLabel="NameAndGeneralInfoSection"
-            updatePath="/personal-information/name-and-general-information"
-            component={<NameAndGeneralInfoSection reviewSection/>}/>
+            sectionLabel="PersonalInfoSection"
+            updatePath="/veteran-information/personal-info"
+            component={<PersonalInfoSection reviewSection/>}/>
+
+        <ReviewCollapsiblePanel
+            sectionLabel="DemographicInformationSection"
+            updatePath="/veteran-information/demographic-information"
+            component={<DemographicInformationSection reviewSection/>}/>
 
         <ReviewCollapsiblePanel
             sectionLabel="VAInformationSection"
-            updatePath="/personal-information/va-information"
+            updatePath="/veteran-information/demographic-information"
```

Comment:
I am leaving all things `<ReviewAndSubmitSection/>` to the `review-section-cleanup` PR. Which is coming right after this one.